server/src/repositories/metadata.repository.ts

```diff
@@ -60,49 +60,42 @@ export class MetadataRepository implements IMetadataRepository {
 
   @GenerateSql({ params: [DummyValue.UUID] })
   async getCountries(userId: string): Promise<string[]> {
-    const entity = await this.exifRepository
+    const results = await this.exifRepository
       .createQueryBuilder('exif')
       .leftJoin('exif.asset', 'asset')
       .where('asset.ownerId = :userId', { userId })
-      .andWhere('exif.country IS NOT NULL')
-      .select('exif.country')
+      .select('exif.country', 'exif_country')
```

Comment:
Is there a particular reason you set an alias here and make it a raw query? You can remove the alias and use `getMany` like the original, just removing the `filter` at the end.

Reply:
Right, when I did that it only returned one result (United States) instead of two (United States, null).

Reply:
Oh, maybe because you don't select the `id` field. That's fine, it can be a raw query. Maybe just make the alias camel case (or remove it).